students/19th/jongholee/style/login.css

```
@@ -0,0 +1,146 @@
+.login_flex{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+
+    width: 100vw;
+    height: 100vh;
+
+    background-color: #FAFAFA;
+}
+.login_container{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+
+    width: 350px;
+    height: 380px;
+
+    border: 1px solid rgba(var(--b6a,219,219,219),1);
+    border-radius: 1px;
+
+    background-color: white;
+}
+
+.login_title{
+    margin-bottom: 5px;
+
+    font-size: 39px;
+    font-family: 'Lobster', cursive;
+}
+
+.login_form{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-content: center;
+
+    width: 348px;
+    height: 211px;
+}
+
+.login_id_div, .login_pw_div {
+    width: 268px;
+    height: 36px;
+
+    margin: 5px auto;
+}
+
+.login_id_span, .login_pw_span {
+    position: absolute;
+    width:  267px;
+    height: 36px;
+
+    margin-top: 5px;
+    margin-left: 1px;
+
+    padding-top: 12px;
+    padding-left: 9px;
+
+    font-size: 12px;
+    color: #999999;
+    background-color: #FAFAFA;
+    border-radius: 3px;
+}
+#login_id, #login_pw{
+    position: relative;
+
+    width: 268px;
+    height: 36px;
+
+    margin: 5px auto;
+    padding: 8px 0px 5px 7px;
+
+    border: 1px solid #EFEFEF;
+    border-radius: 3px;
+    color: #999999;
```

종호님 컬러 6자리가 모두 같은 숫자일 경우 
```suggestion
    color: #999;
```
이렇게도 가능합니다!